5.5/test/run

```diff
@@ -140,6 +140,7 @@ function run_replication_test() {
       echo "The ${slave_ip} failed to register in MASTER"
       return 1
     fi
+    docker logs $(get_cid slave.cid)
```

Comment:
is this critical as status/progress information, or can you make this only dump out when there's a failure?  we already dump container logs in cleanup() on failures, but not the slave container currently of course.